core/src/main/java/io/questdb/cutlass/text/TextImportExecutionContext.java

```diff
@@ -46,15 +48,19 @@ public AtomicBooleanCircuitBreaker getCircuitBreaker() {
     }
 
     public boolean isActive() {
-        return activeImportId.get() > -1;
+        return activeImportId.get() > NO_IMPORT_ACTIVE;
     }
 
     public boolean sameAsActiveImportId(long importId) {
```

Comment:
nit: probably we don't need this method anymore - there is `getCurrentImportId()` method which returns the id.